src/app/components/Results/ResultsList.jsx

```diff
@@ -20,12 +20,23 @@ class ResultsList extends React.Component {
     this.getRecord = this.getRecord.bind(this);
   }
 
+  /*
+   * getRecord(e, bibId, itemId)
+   * @description Get updated information for an item along with its delivery locations,
+   * and the route to the correct page.
+   * @param {object} e Event object.
+   * @param {string} bibId The bib's id.
+   * @param {string} itemId The item's id.
+   */
   getRecord(e, bibId, itemId = '') {
     e.preventDefault();
 
-    ajaxCall(`/api/bib?bibId=${bibId}`,
+    ajaxCall(`/api/hold/request/${bibId}-${itemId}`,
```

Comment:
This isn't right. You may need to create another function for this functionality. If you check the `if` statement below, if there's an `itemId` then it will route to the hold request page, and if there's no `itemId` it will route to the bib page. You only need the bib's information and you don't need delivery locations or the edd requestable flag just for the bib page.

So you can create a new function to handle the two cases and make sure you follow it correctly in the `<ItemTable>` component.